### ISSUE TITLE
Can't disable notifications permission prompt using kOSSettingsKeyAutoPrompt on iOS 11

### ISSUE THREAD

**Issue by umarsheikh13** [OneSignal/OneSignal-Cordova-SDK#337]
Hey guys

I'm having trouble disabling the notifications prompt using the "kOSSettingsKeyAutoPrompt" setting. I've done a lot of troubleshooting on this so I'm not sure what I'm missing.

I built a very simple app with just the code below but it still prompts me for permission on app start.

```
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="minimal-ui, viewport-fit=cover, initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
        <meta http-equiv="X-UA-Compatible" content="ie=edge">
        <title>App</title>
    </head>
    <body>
        <script type="text/javascript" src="cordova.js"></script>
        <script>
            document.addEventListener('deviceready', function() {
                window.plugins.OneSignal
                    .startInit('APP_ID')
                    .iOSSettings({
                        kOSSettingsKeyAutoPrompt: false,
                        kOSSettingsKeyInAppLaunchURL: false
                    })
                    .endInit();
            }, false);
        </script>
    </body>
</html>
```

My app details: (I'm using phonegap build)

cli-8.0.0
iOS 4.5.4
plugin 2.3.2

Any help would be much appreciated.

**Comment by umarsheikh13** [comment]
Looks like the issue is with 2.3.2, more specifically the OneSignal iOS SDK. I've reverted back to version 2.3.1 which works perfectly fine.

```
<plugin name="onesignal-cordova-plugin" source="npm" version="2.3.1" />
```

**Comment by Nightsd01** [comment]
@umarsheikh13 This issue has been resolved in 72de4b7 and will be included in the next release, which I expect to happen in the next few days.

**Comment by umarsheikh13** [comment]
Awesome, thanks @Nightsd01
